### DISCUSSION TITLE
İBAX.İO SPONSORLARI KİMLER?

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#651] [Soru Cevap - Türkçe] [answered]
İbax.io adına tanınmış kişiler veya platform veya şirket olarak kimler sponsor?

**Comment by Yusufknk** [comment, marked as answer]
İbax anasayfasinda Resources kısmından ibax takımında kimler olduğu ile ilgili bil alabilirsiniz. Ayrıca Coinmarketcap ve Coingecko'da listelenecektir.

**Reply by Akrepkapo** [reply]
Teşekkürler
